internal/cloud/realm/secrets_ext_test.go

```diff
@@ -0,0 +1,50 @@
+package realm_test
+
+import (
+	"testing"
+
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	u "github.com/10gen/realm-cli/internal/utils/test"
+	"github.com/10gen/realm-cli/internal/utils/test/assert"
+
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+func TestSecrets(t *testing.T) {
+	u.SkipUnlessRealmServerRunning(t)
+
+	t.Run("should fail without an auth client", func(t *testing.T) {
+		client := realm.NewClient(u.RealmServerURL() + "")
+
+		_, err := client.Secrets(primitive.NewObjectID().Hex(), primitive.NewObjectID().Hex())
+		assert.Equal(t, realm.ErrInvalidSession{}, err)
+	})
+
+	t.Run("with an active session ", func(t *testing.T) {
+		client := newAuthClient(t)
+		groupID := u.CloudGroupID()
+
+		secretName := "secretName"
+		secretValue := "secretValue"
+
+		testApp, teardown := setupTestApp(t, client, groupID, "secrets-test")
+		defer teardown()
```

Comment:
@nathantfrank FYI once this is merged, these 2 lines can be used in the user_ext test. If you merge first, I can add them in there for `user revoke`